astropy/io/fits/util.py

```diff
@@ -26,13 +26,8 @@
 
 from astropy.utils.exceptions import AstropyUserWarning
 
-try:
-    # Support the Python 3.6 PathLike ABC where possible
-    from os import PathLike
-    path_like = (str, PathLike)
-except ImportError:
-    path_like = (str,)
-
+from os import PathLike
```

Comment:
@saimn might want to double check. I think `PathLike` is available since Python 3.4.

Reply:
Seems good, it was added in 3.6 : https://docs.python.org/3/library/os.html#os.PathLike